### ISSUE TITLE
When will llama.cpp be updated with the latest ROCm/hip related content?

### ISSUE THREAD

**Issue by amdjiahangpan** [XiaoMi/xiaomi-miloco#165]
After compiling, I found that loading all 37 layers of MiMo-VL-Miloco-7B_Q4_0.gguf resulted in NaN values ​​for logits, while loading 36 layers allowed it to run.

Could you provide me with the official development process and compilation scripts?

I'm using rocm 7.1.0.
